doc/source/installation/apple_arm.rst

```diff
@@ -22,11 +22,16 @@ To install the dependencies (mpi, p4est, trilinos and METIS) all together using
 .. code-block:: text
   :class: copy-button
 
-  sudo apt-get install libmuparser-dev
+  brew install muparser
 
 Clone the candi git repository in a folder of your choice  (e.g. ``$HOME/software/``). You can edit the ``candi.cfg`` file if you want to force the installation of the deal.II master version instead of the current stable version by setting ``DEAL_II_VERSION=master`` on line 97. Under Apple ARM, we only recommend the installation of the required libraries, namely parmetis, trilinos and p4est.
```

Comment:
This is what I was talking to you about, it was already mentioned in the documentation to install only the three required packages